meta-luneos/recipes-multimedia/alsa/alsa-ucm-conf/0006-Add-UCM-for-RK817.patch

```diff
@@ -0,0 +1,129 @@
+From b0497ca4b508d6f894d78b16e0e06616a2a36c16 Mon Sep 17 00:00:00 2001
+From: Chris Morgan <macromorgan@hotmail.com>
+Date: Fri, 24 Feb 2023 13:33:20 -0600
+Subject: [PATCH] ucm2: Rockchip: rk817: Add ALSA UCM support
+
+Add ALSA-UCM support for the Rockchip RK817 audio codec. This codec
+is typically configured either with an internal or external amplifier
+as reflected by the longname. This configuration has been tested on
+the Anbernic RG353P (rk817_ext) and the Odroid Go Advance (rk817_int).
+
+Changes from V1:
+ - Use a constant to control if the output mux is set to HP or SPK.
+
+Link: https://lore.kernel.org/alsa-devel/20230224193320.1503-1-macroalpha82@gmail.com/
+Signed-off-by: Chris Morgan <macromorgan@hotmail.com>
+Signed-off-by: Jaroslav Kysela <perex@perex.cz>
```

Comment:
Add Upstream-Status

Reply:
Sorry need to get used to this new habit ;)